examples/plotting/xy_lims.py

```diff
@@ -41,11 +41,8 @@
 # determine which pixel coordinates these represent on the Map.
 
 world_coords = SkyCoord(Tx=xlims_world, Ty=ylims_world, frame=aia_map.coordinate_frame)
-pixel_coords = aia_map.world_to_pixel(world_coords)
+pixel_coords_x , pixel_coords_y = aia_map.wcs.world_to_pixel(world_coords)
```

Comment:
You will need to update the lines below that set the xlim and ylim otherwise this example will error.

Reply:
That is surprising to me, if that's the case, then that's ok.